watertap/unit_models/electrodialysis_1D.py

```diff
@@ -103,11 +117,56 @@ class Electrodialysis1DData(UnitModelBlockData):
     CONFIG.declare(
         "operation_mode",
         ConfigValue(
-            default="Constant_Current",
-            domain=In(["Constant_Current", "Constant_Voltage"]),
+            default=ElectricalOperationMode.Constant_Voltage,
+            domain=In(ElectricalOperationMode),
             description="The electrical operation mode. To be selected between Constant Current and Constant Voltage",
         ),
     )
+    CONFIG.declare(
+        "limiting_current_density_method",
+        ConfigValue(
+            default=LimitingCurrentDensityMethod.InitialValue,
+            domain=In(LimitingCurrentDensityMethod),
+            description="Configuration for method to compute the limiting current density",
+            doc="""
+           **default** - ``LimitingCurrentDensityMethod.InitialValue``
+
+       .. csv-table::
+           :header: "Configuration Options", "Description"
+
+           "``LimitingCurrentDensityMethod.InitialValue``", "Limiting current is calculated from a single initial value of the feed solution tested by the user." 
+           "``DensityCalculation.seawater``", "Limiting current density is caculated from the empirical equation: TODO "
+           "``DensityCalculation.laliberte``", "Limiting current density is calculated from a theoretical equation: TODO "
+       """,
+        ),
+    )
+
+    CONFIG.declare(
+        "limiting_current_density_data",
+        ConfigValue(
+            default=500,
+            description="Limiting current density data input",
+        ),
+    )
```

Comment:
Why set this as a config option when if could just be a variable in the model that user could fix?

Reply:
This config would only be used when the `InitialValue` method (the other config) is selected.  The actual "i_lim" is a x-dependent var in the 1-d model and is calculated from this single data from user input. Adding it as a var will cause larger changes in the mode structure when adding other methods.   This way also makes the scaling easy.  So I feel it would be efficient to set it up like this.